### PR TITLE
Update stats.binom_test() docstring

### PR DESCRIPTION
**Pull request by srvanrell**
Add description for parameter `alternative` in the `stats.binom_test()` docstring.

### CODE REVIEW

**Comment by gfyoung** [comment]
In your commit message, write something like "Closes gh-5833" so that your issue will be automatically closed when this PR gets merged.

**Comment by srvanrell** [comment]
@gfyoung where should I write that? in the previous commit or in a new one?

**Comment by ev-br** [comment]
The easiest way is to edit the incantation into the text of the PR message.

File: scipy/stats/morestats.py [hunk 1]
```diff
@@ -1994,6 +1994,8 @@ def binom_test(x, n=None, p=0.5, alternative='two-sided'):
     p : float, optional
         The hypothesized probability of success.  0 <= p <= 1. The
         default value is p = 0.5
+    alternative : {'two-sided', 'greater', 'less'}, optional
+        indicates the alternative hypothesis.
```
Comment (ev-br) [inline scipy/stats/morestats.py]: For optional arguments please state the default. And while you're at it, please make "indicate" start with a capital letter.

**Comment by srvanrell** [comment]
@ev-br what is the incantation? I'm not familiar with pull requests :/

**Comment by ev-br** [comment]
https://help.github.com/articles/closing-issues-via-commit-messages/

This is for your future PRs, not necessarily for this one :-).

**Comment by srvanrell** [comment]
Thanks!

**Comment by ev-br** [comment]
Squashed and merged as https://github.com/scipy/scipy/commit/2c17f133e7fa889177bc0ac473930795a845c067

Thanks @srvanrell and congratulations with what I believe is your first scipy commit. Keep them coming!

**Pull request closed without merging**